crates/driver/src/domain/competition/solution/fee.rs

```diff
@@ -126,7 +126,11 @@ impl Fulfillment {
                 )?;
                 self.calculate_fee(price_limits, prices, *factor, *max_volume_factor)
             }
-            FeePolicy::Volume { factor } => self.fee_from_volume(prices, *factor),
+            FeePolicy::Volume { factor } => {
+                let fee_from_volume = self.fee_from_volume(prices, *factor)?;
+                tracing::debug!(uid=?self.order().uid, ?fee_from_volume, executed=?self.executed(), surplus_fee=?self.surplus_fee(), "calculated protocol fee");
```

Comment:
`cargo fmt` is not always able to format long macros.
Could you manually add the required line breaks here and for the `price improvement` log?

Reply:
cargo fmt returns errors when i manually add breaks